### PR TITLE
Ignore www subdomains

### PR DESCRIPTION
**Pull request by fivism**
When checking our own stash of parsers, strip `www`'s from the parsed url
netloc.

Straightforward way of dealing with #4 without bringing regex's into the equation?

### CODE REVIEW

**Comment by fivism** [comment]
Alternatively add https://pypi.org/project/tldextract/ as a dependency for greater confidence

File: main.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def scrape_recipe(url):
 
     if not recipe:
         parsed_uri = urllib.parse.urlparse(url)
-        domain = parsed_uri.netloc.lower()
+        domain = parsed_uri.netloc.lower().replace('www.', '', 1)
```
Comment (poundifdef) [inline main.py:27]: Nice! How would you feel about adding a little check to make sure this is only present at the beginning of the domain? eg, `if domain.startswith('www.')` then perform the replace operation?

**Comment by fivism** [comment]
[This](https://github.com/fivism/plainoldrecipe/commit/be7ccd2ee7c97b866557d68924829ee7069289f8) is also what it could look like using tldextract instead of urllib. Bit more explicit

File: main.py (hunk 1 above)
Comment (fivism) [reply main.py]: Would feel great about adding a check ;)

**Comment by poundifdef** [comment]
Thank you for making the change! I like the _idea_ of tldextract, however, I am hesitant to add a library which performs another HTTP request when fetching (this library makes an external call to get a list of valid TLDs.) It also saves a local cached copy of valid TLDs, but I haven't explored what local file storage looks like on google app engine.

Otherwise, happy to have the simple fix that you've implemented, and I'll go ahead and merge!

**Comment by fivism** [comment]
`It also saves a local cached copy of valid TLDs, but I haven't explored what local file storage looks like on google app engine.`

Definitely overlooked this. Thanks!

**Pull request merged**